examples/hipaa/variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "region" {
+  type        = string
+  description = "AWS Region"
+}
+variable "create_iam_role" {
+  description = "Flag to indicate whether an IAM Role should be created to grant the proper permissions for AWS Config"
+  type        = bool
+  default     = false
+}
+
+variable "create_sns_topic" {
+  description = <<-DOC
+    Flag to indicate whether an SNS topic should be created for notifications
+    If you want to send findings to a new SNS topic, set this to true and provide a valid configuration for subscribers
+  DOC
+
+  type    = bool
+  default = false
+}
+
+variable "force_destroy" {
+  type        = bool
+  description = "(Optional, Default:false ) A boolean that indicates all objects should be deleted from the bucket so that the bucket can be destroyed without error. These objects are not recoverable"
```

Comment:
Docs will already tell us the default and whether it is optional. Do we want the extra text here?

Reply:
082d3070ec9fda6122700430df3f2e78214edbd4